src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -138,6 +138,18 @@ class IndexCollectionManager(
     }
   }
 
+  override def getIndex(indexName: String, logVersion: Int): Option[IndexLogEntry] = {
+    withLogManager(indexName) { logManager =>
+      logManager.getLog(logVersion).map(_.asInstanceOf[IndexLogEntry])
+    }
+  }
+
+  override def getAvailableIndexVersions(indexName: String): Seq[Int] = {
+    withLogManager(indexName) { logManager =>
+      logManager.getIndexVersions(Seq(Constants.States.ACTIVE))
```

Comment:
Actually, can we have `def getIndexVersions(states: Seq[String]): Seq[Int]` instead of `getAvailableIndexVersions`? "Available" <=> "ACTIVE" doesn't seem to match nicely. (sorry for the wrong suggestion in the beginning). The caller `closestIndex` can decide which state it wants to retrieve.